### DISCUSSION TITLE
手机部分app不明原因接不上，但另外一些app可以

### DISCUSSION THREAD

**Discussion by edsin1011** [2dust/v2rayNG#3226] [General]
本人用的搬瓦工官网服务，然后接上v2rayng，google play商城和youtube经常毫无反应，但同时浏览器登录没有问题，或者其他app例如x或者fb正常。不知道什么原因

**Comment by Oraclediy** [comment]
解决了吗，我也是这个问题，同一个节点，手机浏览器不能使用，但是电报可以使用。
